http/source_service.go

```diff
@@ -226,7 +226,13 @@ func (h *SourceHandler) handleGetSourcesBuckets(w http.ResponseWriter, r *http.R
 		return
 	}
 
-	bs, _, err := h.BucketService.FindBuckets(ctx, req.filter)
+	_, err = h.SourceService.FindSourceByID(ctx, req.getSourceRequest.SourceID)
```

Comment:
I've been out of the loop for a while. Is this route expected to work? I thought we were rethinking how sources would work.

Reply:
yeah, i am not aware of any new plans.